cpp/include/ray/core.h

```diff
@@ -0,0 +1,10 @@
+
+#pragma once
+
+#include "ray/common/buffer.h"
+#include "ray/common/function_descriptor.h"
+#include "ray/common/id.h"
+#include "ray/common/task/task_common.h"
+#include "ray/common/task/task_spec.h"
+#include "ray/common/task/task_util.h"
+#include "ray/core_worker/context.h"
```

Comment:
I prefer deleting this file and including individual header file on demand.

Reply:
The benefit of this way is a clearer dependencies between c++ worker and core? Maybe we can rethink this after implement cluster mode?